index.js

```diff
@@ -1,81 +1,143 @@
-const miio = require('miio')
-let Service, Characteristic
+const miio = require('miio');
```

Comment:
Can you remove the semicolons. I don't use it because it's add no information in the code and it will be easier to see your change on the PR.

Reply:
Removed.